src/workerd/jsg/modules.h

```diff
@@ -437,70 +437,70 @@ class ModuleRegistryImpl final: public ModuleRegistry {
     entries.insert(kj::heap<Entry>(specifier, Type::BUNDLE, kj::fwd<ModuleInfo>(info)));
   }
 
+  void addBuiltinModule(Module::Reader module) {
```

Comment:
What's this refactor? Did anything change in this code?

Reply:
Nope nothing changed, separated it out so that we could call this directly for the one `Module` in the bundle that is the `node:async_hooks`